### PR TITLE
make surface flux of density consistent

### PR DESCRIPTION
**Pull request by szy21**

### CODE REVIEW

**Review by jiahe23 — APPROVED** [review]

File: examples/hybrid/sphere/baroclinic_wave_utilities.jl [hunk 1]
```diff
@@ -422,5 +422,6 @@ function vertical_diffusion_boundary_layer_tendency!(Yₜ, Y, p, t)
             bottom = Operators.SetValue(-mean(dif_flux_ρq_tot)),
         )
         @. Yₜ.c.ρq_tot += ᶜdivᵥ(ᶠK_E * ᶠinterp(ᶜρ) * ᶠgradᵥ(Y.c.ρq_tot / ᶜρ))
+        @. Yₜ.c.ρ += ᶜdivᵥ(FT(0) * ᶠgradᵥ(ᶜρ))
```
Comment (charleskawczynski) [inline examples/hybrid/sphere/baroclinic_wave_utilities.jl:425]: Isn't this just adding zero?

Comment (szy21) [reply examples/hybrid/sphere/baroclinic_wave_utilities.jl:425]: It has a bottom boundary condition though. Basically we want only surface flux to density, but not vertical diffusion in the interior.

**Pull request merged**